docs/deployment_guide/partner_editable/architecture.adoc

```diff
@@ -20,11 +20,11 @@ As shown in <<architecture1>>, this Quick Start sets up the following:
 *** An Amazon Elastic Compute Cloud (Amazon EC2) Linux instance to bootstrap the installation of VMware TAP in an Amazon EKS cluster and serve as administrator bastion host.
 *** An Amazon EC2 Windows bastion host for access to the Tanzu Application Platform user interface.
 * In the private subnets:
-** A group of Amazon EKS nodes in an Auto Scaling Group with Contour Envoy ingress controller pods and the TAP installation. For a complete list of the {partner-product-short-name} packages, refer to https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.2/tap/GUID-install-components.html#verify-the-installed-packages-1[Verify the installed packages].
+** A group of Amazon EKS nodes in an Auto Scaling Group with Contour Envoy ingress controller pods and the TAP installation. For a complete list of the {partner-product-short-name} packages, refer to https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.3/tap/GUID-install-components.html#verify-the-installed-packages-1[Verify the installed packages].
 ** A Network Load Balancer (NLB) attached to the private subnets to direct traffic to the Contour Envoy pod. The Contour Envoy container provides ingress and reverse proxy to the TAP pods for the TAP user interface and metadata store.
 * Amazon CloudWatch for storing Linux bastion host performance metrics and TAP bootstrap logs.
```

Comment:
```suggestion
* Amazon CloudWatch for storing Linux bastion host metrics and TAP bootstrap logs.
```

Reply:
TODO: duplicate in the landing page afterward.